test/tests/geomsearch/3d_penetration_locator/tests

```diff
@@ -33,4 +33,11 @@
                   "point search leads to element Jacobian computations too far from the element for "
                   "the Jacobian to be non-negative."
   []
+  [close_tet_mesh]
+    type = Exodiff
+    input = 'close_tet.i'
+    exodiff = 'close_tet_out.e'
+    recover = false # steady solve
+    requirement = "The system will shall be able to ensure contact points are within tangential tolerances."
```

Comment:
```suggestion
    requirement = "The system shall be able to ensure contact points are within tangential tolerances."
```

Reply:
Actually, shouldn't it say "The system shall ensure contact points..."? We've been trying to avoid saying "be able to".